examples/mockups/pricing.html

```diff
@@ -0,0 +1,786 @@
+    <script src="/dist/js/site.js"></script>
+  </head>
+  <body id="pricing">
+    
+    <!-- Top Menu -->
+    <div class="ui menu fixed borderless" id="topnav">
+      <div class="ui container">
+
+        <a class="header item" href="">
+          <div class="ui image readthedocs-logo"><img src="./images/logo-wordmark-dark.svg"/></div>
+        </a>
+
+        <a class="item active" href="">Product</a>
+        <a class="item" href="">Pricing</a>
+        <a class="item" href="">Company</a>
+        <a class="item" href="">Resources</a>
+    
+        <div class="right menu">
+          <a class="item" href="">Support</a>
+          <div class="item">
+            <a class="ui button primary">
+              Go to Application
+            </a>
+          </div>
+        </div>
+    
+      </div>
+    </div>
+    <!-- end Top Menu -->
+
+
+    <!-- Contents 
+    ----------------------------------- -->
+
+    <!-- Intro -->
+    <section class="ui vertical segment very padded" id="intro">
+      <div class="ui container center aligned">
+        <h1 class="ui header massive">Intro</h1>
+        <p>
+          Some intro explaining stuff.
+        </p>
+      </div>
+    </section>
+
+    <!-- Tables -->
+    <section class="ui vertical segment light very padded" id="tables">
+      <div class="ui container">
+        
+        <div class="ui three stackable cards">
+          
+          <!-- Card -->
+          <div class="ui card">
+
+            <div class="content">
+              
+              <div class="ui label ribbon blue big">Base</div>
+
+              <div class="ui header huge">Free</div>
+              <div class="meta">
+                <span class="category">forever</span>
+              </div>
+
+              <button class="ui button fluid large blue">Subscribe</button>
+
+              <div class="description">
+                
+                <div class="ui list relaxed divided">
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Automatic build</div>
+                      <div class="description">on every commit</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Multi-version handling</div>
+                      <div class="description">from branches or tags</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Downloadable formats</div>
+                      <div class="description">PDF, HTML</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Enhanced full-text search</div>
+                    </div>
+                  </div>
+                </div>
```

Comment:
I was a bit torn too on how to show the lists and my ideas here are not set in stone. Also, a lot can change based on the amount and density of information that we may define for this page.

Presently, these are my main thoughts and the reason I choose to go this way. As I see it, we have 3 main options for the pricing table layout:

#### Option 1: The "All that's included in previous, plus..." option
This seems to be the new trend. On first sight, it looks to the user that they won't have a lot to read and that they will be able to skim through it really fast. When the user starts reading however, is sent on a goose hunt. It makes me think of those old multi-option books where you were sent from chapter to chapter with very little time to focus on the underlying history. For example, in order to learn what the Pro plan contains, the user will start reading there, thinking "ok, let me see what this plan costing X has to offer" on the very first line of the plan, he's sent back however. He now has to look for the title his chosen plan sent him to - most likely the one left to it - he will again skim through the price and subscribe button on the card header and again rich the 1st line. Here he's sent back one more card... He now, getting what we want him to do, reads the 1st card, then moves on to the 2nd and then the 3rd. So, in terms of features, the user had to read the exact same amount, only they are disposed in a more left to right manner instead of top to bottom. The big issues I see here is that he also got a lot more noise while trying to read - with all the different card headers on the way - and he had to do a lot more thinking, not on what we probably want him to spend time on - the plans them selves - but on how to read them.

To resume my feelings on this option: it looks at first sight to offer a much faster and clean reading, but as the user starts  to read each plan, that turns out not to be true. Still, I think it works well for pricings with very few features and where we are looking for a really compact look that does not go into technical specifications.


#### Option 2: Show a complete list of features for each plan disposed in a tabular manner.

Requisites:
* The repeated features for each plan must be aligned. For this to be accomplished:
   * either **a)** unique features come at the end of each list
   * or **b)** plans that don't include some feature must reserve the same space blank.

Pros: 
* Plans are easily compared and skimmed to to find differences.
* Each single plan can be read in one go, independently from any other plan.
* case **a)** important extra features for each plan come at the end of the column being very easy to find and compare.

Cons:
* Features are repeated for each plan and can become clustered.
* Either **a)** important extra features for each plan come at the end of the column.
* or **b)** important extra features for each plan are not so easy to skim through.


**Option 3: Show an additional column with a list of all features, dispose plans as columns and add a checkmark in line with each feature each plan have.**

Requisites:
* Mostly just technical/display requisites: data must be shown in a very strict tabular disposition or it looses all meaning.

Pros:
* Plans are easily compared and skimmed to to find differences.
* Important extra features for each plan are commonly still displayed at the end, and because of this, like in the previous option, are easy to find and compare.

Cons:
* Extremely mobile unfriendly. The way around this would be to revert to option 1 or 2 on mobile, but that would mean we would actively have to maintain two different structures.
* Users still need to read through 2 columns to learn about each plan.
* It's to get lost when trying to map features to columns.
* Important extra features for each plan are commonly still displayed at the end.


There are many smaller variations for each option that we can use to highlight specific views or to enforce specific marketing strategies, but as the feature listing goes I think these are the main structures.

There would also be a fourth option for plans that don't share, or don't care to acknowledge, any common features. I don't think this applies to us.

Reply:
Issue #12 proposes a fifth option, combining option 1 and 2 by expecting input.

Reply:
I think #12 is probably the middle ground that would be the best. What's the interim solution here? Are we going to stick with a truncated list using "Basic plan features, plus:" -- basically what we do now on our pricing page.

Reply:
Yes. The idea is to truncate the list initially and show only the extra feactures for each plan - almost exactly like we have now, the only difference beeing the extra button that allows the truncatd part to be shown.

To execute this the best final approach will probably be to hide with CSS the part of the list to be truncated and then show it again with JS on the button click event.

So in terms of HTML it should remains mostly as it is. Not sure if it makes sense to remove the HTML to be truncated now, before merging, to add it later again after #12 is implemented.